opm/simulators/thresholdPressures.hpp

```diff
@@ -187,7 +187,7 @@ void computeMaxDp(std::map<std::pair<int, int>, double>& maxDp,
             int pvtRegionIdx = pvtRegion[cellIdx];
 
             double T = initialState.temperature()[cellIdx];
-            double p = initialState.pressure()[cellIdx];
+            double p = phasePressure[wpos][cellIdx];
```

Comment:
The thpress calculated using this branch is closer to ecl. I doesn't seems to effect the convergence for the first steps.